jetty-core/jetty-core-server/src/main/java/org/eclipse/jetty/core/server/Server.java

```diff
@@ -684,5 +678,36 @@ public DateField(long seconds, HttpField dateField)
         }
     }
 
-    private static class DynamicErrorHandler extends ErrorHandler {}
+    private static class DynamicErrorProcessor extends ErrorProcessor {}
+
+    private static class ServerRequest extends Request.Wrapper
+    {
+        private ServerRequest(Request wrapped)
+        {
+            super(wrapped);
+        }
+
+        @Override
+        public void accept(Processor processor) throws Exception
+        {
+            getWrapped().accept((rq, rs, cb) ->
+            {
+                try
+                {
+                    processor.process(this, rs, cb);
+                }
+                catch (Throwable x)
```

Comment:
This is a clever "trick" to ensure processors see the correct wrappers.   But it is a little strange that:
 + The ServerRequest is a wrapper applied after customizer wrappers
 + The passed request is ignored and this is passed on.

Maybe should have an assert?

```suggestion
        public void accept(Processor processor) throws Exception
        {
            getWrapped().accept((rq, rs, cb) ->
            {
                assert rq.getWrapper() == this;
                try
                {
                    processor.process(this, rs, cb);
                }
                catch (Throwable x)
```

We are going to have to document things very carefully to ensure that users do this trick in their own handlers.   This API does open up the possiblility of wrappers being lost, different wrappers used for accept vs process and all sorts of other strange edge cases.   I fear some users might find a "use" for such anti-tricks